engine/src/main/scala/cromwell/engine/backend/io/filesystem/gcs/GcsFileSystemProvider.scala

```diff
@@ -95,8 +95,7 @@ class GcsFileSystemProvider private (storageClient: Try[Storage], executionConte
     case gcsPath: NioGcsPath =>
       Try(withRetry(client.objects.get(gcsPath.bucket, gcsPath.objectName).execute)) recover {
         case ex: GoogleJsonResponseException
-          if ex.getStatusCode == 404 =>
-          if (!gcsPath.isDirectory) throw new FileNotFoundException(path.toString)
+          if ex.getStatusCode == 404 => if (!gcsPath.isDirectory) throw new FileNotFoundException(path.toString)
```

Comment:
Yes, that's what the Nio stuff expects - I got confused too at first, I did as you suggested and had a bunch of random errors when manipulating directories